src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -112,6 +112,36 @@ def may_share_memory_mapper(a: Any, b: Any, max_work: Optional[int] = None) -> b
     return False
 
 
+def full_like_mapper(
+    a: Union[pd.DataFrame, pd.Series],
+    fill_value: Hashable,
+    dtype: Optional[Any] = None,
+    order: Optional[str] = "K",
+    subok: Optional[bool] = True,
+    shape: Optional[tuple[Any]] = None,
+) -> Union[pd.DataFrame, pd.Series]:
+    if not subok:
+        return NotImplemented
+    if not order == "K":
+        return NotImplemented
+    if dtype is not None:
+        return NotImplemented
+
+    result_shape = shape
+    if isinstance(result_shape, tuple) and len(result_shape) == 0:
```

Comment:
nit: this won't do the right thing for `np.full_like(pd.Series(1), 4, shape=[])`. I think you want `is_list_like` or something like that.

Reply:
good catch

Reply:
Are you still missing a fix for this place a `shape=[]` test case?

Reply:
I will return a TypeError in this case.